spec/unit/filter_option_parser_spec.rb

```diff
@@ -62,24 +62,24 @@
         File.write(exclude_file, "/foo/bar\n/baz \n")
         filter = subject.parse("inspect", "skip-files" => "/foo,@#{exclude_file}")
         expect(filter.to_array).to match_array([
-          "/unmanaged_files/files/name=/foo",
-          "/unmanaged_files/files/name=/foo/bar",
-          "/unmanaged_files/files/name=/baz"
+          "/unmanaged_files/name=/foo",
```

Comment:
Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.